Add information in documentation about patching by environment variable (#242)
IntelPython/daal4py: doc/sklearn.rst

    python -m daal4py my_application.py

> To enable daal4py patches through an environment variable, you can set the environment variable.

Reviewer: ```suggestion
To enable daal4py patches through an environment variable, set ``USE_DAAL4PY_SKLEARN`` to 1 as shown below.
```

On Linux and Mac OS::

    export USE_DAAL4PY_SKLEARN=1

On Windows::

    set USE_DAAL4PY_SKLEARN=1

> To disable, set the environment variable to 0.

Reviewer: ```suggestion
To disable daal4py patches, set the ``USE_DAAL4PY_SKLEARN`` environment variable to 0.
```

    python -m daal4py my_application.py

> If using scikit-learn from intel conda channel, then 

Reviewer: ```suggestion
If you are using scikit-learn from Intel conda channel, then 
```

wouldn't `Intel(R) Distribution for Python` be a correct way to name "scikit-learn from Intel conda channel"?

Author: This is a good question, I will ask about it

Author: Fixed

> you can enable daal4py patches through an environment variable, set ``USE_DAAL4PY_SKLEARN`` to one of the values
> ``True``, ``'1'``, ``'y'``, ``'yes'``, ``'Y'``, ``'YES'``, ``'Yes'``, ``'true'``, ``'True'``, ``'TRUE'`` as shown below.

Reviewer: ```suggestion
you can enable daal4py patches through an environment variable. To do this, set ``USE_DAAL4PY_SKLEARN`` to one of the values
``True``, ``'1'``, ``'y'``, ``'yes'``, ``'Y'``, ``'YES'``, ``'Yes'``, ``'true'``, ``'True'``, or ``'TRUE'`` as shown below.
```

Author: Fixed
